### ISSUE TITLE
Gitcoin grants: Email blast to previous funders displays wonky

### ISSUE THREAD

**Issue by AnneConnelly** [gitcoinco/web#8010]
**Describe the bug**
When you use the eblast function to send an update to funders, the email looks bad. 

**To Reproduce**
Send an update and click the email grant funders box.

**Expected behavior**
That the email would look professional.

**Screenshots**

<img width="936" alt="email1" src="https://user-images.githubusercontent.com/16226861/100936234-3525ce00-34bf-11eb-8cdf-497f22537158.PNG">
<img width="926" alt="email2" src="https://user-images.githubusercontent.com/16226861/100936233-348d3780-34bf-11eb-84aa-5393d04867da.PNG">
<img width="918" alt="email3" src="https://user-images.githubusercontent.com/16226861/100936237-3525ce00-34bf-11eb-97aa-164dec4740b3.PNG">
<img width="944" alt="email4" src="https://user-images.githubusercontent.com/16226861/100936235-3525ce00-34bf-11eb-9298-a82679e8ac82.PNG">

**Desktop (please complete the following information):**

- OS:
- Browser:
- Browser Version:

**Smartphone (please complete the following information):**

- Device: 
- OS: 
- Browser:
- Version:

**Additional context**

**Comment by owocki** [comment]
thanks @AnneConnelly you said on twitter that the email you screencapped is on MS outlook right?

**Comment by AnneConnelly** [comment]
Correct. MS Outlook on a laptop. I can forward the email if you want - for some reason when I went to print the whole email to pdf, it wasn't displaying the way it looked to the recipient, so I had to do awkward screenshots.

**Comment by abitrolly** [comment]
Which Outlook is this? Looks like supporting everything <= 2016 just doesn't worth the effort.
